### PR TITLE
Tests for pair printing & sparsevec

### PR DESCRIPTION
**Pull request by kshyatt**
Sparsevec’s `scale` and `{c}transpose` methods needed tests, as did printing of `Pair`s.

### CODE REVIEW

File: test/show.jl [hunk 1]
```diff
@@ -303,6 +303,11 @@ function f13127()
 end
 @test f13127() == "f"
 
+a = Pair(1.0,2.0)
```
Comment (tkelman) [inline test/show.jl]: use a `let`, and `sprint(show, a)`

**Comment by kshyatt** [comment]
Looks like an unrelated parallel failure. This ok now?

**Comment by tkelman** [comment]
restarted just to be on the safe side

**Comment by tkelman** [comment]
first commit here should be backport-worthy, not the second though since 0.4 doesn't have SparseVector

**Pull request merged**